packages/create-mc-app/src/cli.ts

```diff
@@ -0,0 +1,109 @@
+import { cac } from 'cac';
+import { Listr, ListrTask } from 'listr2';
+import * as tasks from './tasks';
+import { throwIfNodeVersionIsNotSupported } from './validations';
+import { shouldUseYarn } from './utils';
+import hintOutdatedVersion from './hint-outdated-version';
+import processOptions from './process-options';
+import { TCliCommandOptions } from './types';
```

Comment:
nit: 
```suggestion
import type { TCliCommandOptions } from './types';
```

Reply:
`import { type TCliCommandOptions } from './types';` would also work in newer TS versions so you can mix types and regular imports.

Reply:
Thanks @kark.

@tdeekens If we're only importing types, we kind of prefer to use `import type`, as otherwise you need to declare the `type` for every named import.